sdk/core/core-http/package.json

```diff
@@ -115,14 +115,16 @@
     "@azure/core-tracing": "1.0.0-preview.2",
     "@types/node-fetch": "^2.5.0",
     "@types/tunnel": "^0.0.1",
+    "buffer": "^5.2.1",
```

Comment:
I totally could have missed this, but did we start using buffer somewhere new or does this fix something? Curious why this was added as a dependency.

Reply:
Marking as resolved as this change is not part of the final set of changes in this PR